### PR TITLE
compatiblity: expose LOG_ properties globally

### PR DESCRIPTION
**Pull request by sam-github**
Undocumented, but exposed like this for compatability with node-syslog.

connected to connected to strongloop-internal/scrum-nodeops#893

see https://github.com/strongloop-internal/scrum-nodeops/issues/893#issuecomment-136826046

### CODE REVIEW

**Comment by sam-github** [comment]
@rmg this doesn't appear to be in CI, but it was added in https://github.com/strongloop-internal/ci-infrastructure/pull/7/files

PTAL

File: test/test-compat.js [hunk 1]
```diff
@@ -18,5 +18,8 @@ tap.test(function(t) {
   t.doesNotThrow(function() {
     Syslog.close();
   }, 'Syslog.close');
+  t.assert('LOG_KERN' in Syslog, 'facilities are global for compat');
```
Comment (rmg) [inline test/test-compat.js]: Should this also be asserting that it is equal to the other location?

Comment (rmg) [inline test/test-compat.js]: Also, they're not really global ;-)

**Comment by rmg** [comment]
Couple minor nits.

Nothing blocking if you are in a hurry, so LGTM.

File: test/test-compat.js (hunk 1 above)
Comment (sam-github) [inline test/test-compat.js]: fixed

**Pull request merged**